aws/devnet/main.tf

```diff
@@ -2,17 +2,18 @@ module "devnet" {
   source          = "../network-primitives"
   path_to_scripts = "../network-primitives/scripts"
   network_name    = "devnet"
+
   bootstrap-node-config = {
     instance-type      = var.instance_type
-    deployment-version = 0
+    deployment-version = 1
     regions            = var.aws_region
     instance-count     = var.instance_count
     docker-org         = "subspace"
-    docker-tag         = "snapshot-2023-jun-08-2"
+    docker-tag         = "snapshot-2023-aug-15"
     reserved-only      = false
     prune              = false
-    genesis-hash       = "8a94e770a50298516444025b8c075034f8be7745812358d25189175fe504fc66"
```

Comment:
do we not need this anymore ?

Reply:
it's currently set to empty string. Since we gather that value after launching the network on first run.